packages/cloudflare/src/cli/build/patches/plugins/fetch-cache-wait-until.ts

```diff
@@ -0,0 +1,42 @@
+import { patchCode } from "../ast/util.js";
+import type { ContentUpdater } from "./content-updater.js";
+
+/**
+ * The following Next.js code sets values in the incremental cache for fetch calls:
+ *  https://github.com/vercel/next.js/blob/e5fc495e3d4/packages/next/src/server/lib/patch-fetch.ts#L690-L728
+ *
+ * The issue here is that this promise is never awaited in the Next.js code (since in a standard node.js server
+ * the promise will eventually simply just run) but we do need to run it inside `waitUntil` (so that the worker
+ * is not killed before the promise is fully executed), without that this promise gets discarded and values
+ * don't get saved in the incremental cache.
+ *
+ * This function wraps the promise in a `waitUntil` call (retrieved from `globalThis.__openNextAls.getStore()`).
+ */
+export function patchFetchCacheSetMissingWaitUntil(updater: ContentUpdater) {
+  return updater.updateContent(
+    "patch-fetch-cache-set-missing-wait-until",
+    { filter: /\.(js|mjs|cjs|jsx|ts|tsx)$/ },
+    ({ contents }) => {
+      if (/Failed to set fetch cache/.test(contents)) {
```

Comment:
You can use the contentFilter that was recently merged to simplify further

Reply:
Oh wait, I wasn't merged?

Reply:
Just pushed https://github.com/opennextjs/opennextjs-cloudflare/pull/365

You can review and merge then use.

It shouldn't block this PR, we can merge and update later

Reply:
done 🙂